packages/app/src/cli/services/app-logs/ui/components/Logs.tsx

```diff
@@ -0,0 +1,207 @@
+import {AppEventData, subscribeProcess} from '../../../../services/logs.js'
+import {DeveloperPlatformClient} from '../../../../utilities/developer-platform-client.js'
+import React, {FunctionComponent, useRef, useState, useEffect} from 'react'
+
+import {Static, Box, Text} from '@shopify/cli-kit/node/ink'
+
+export interface LogsProps {
+  logsProcess: ({
+    jwtToken,
+    cursor,
+    filters,
+  }: {
+    jwtToken: string
+    cursor?: string | undefined
+    filters?: {
+      status?: string
+      source?: string
+    }
+  }) => Promise<{
+    cursor?: string | undefined
+    errors?: string[] | undefined
+    appLogs?: AppEventData[] | undefined
+  }>
+  developerPlatformClient: DeveloperPlatformClient
+  storeId: string
+  apiKey: string
+  cursor: string
+  jwtToken: string
+  filters?: {
+    status?: string
+    source?: string
+  }
+}
+
+interface DetailsFunctionRunLogEvent {
```

Comment:
I think this is one part that we should go out of our way to share between the commands

Reply:
+1